contracts/AcceleratingDistributor.sol

```diff
@@ -182,6 +182,7 @@ contract AcceleratingDistributor is ReentrancyGuard, Ownable, Multicall {
      * @param amount The amount of the token to stake.
      */
     function stake(address stakedToken, uint256 amount) external nonReentrant onlyEnabled(stakedToken) {
+        require(amount > 0, "Invalid amount");
```

Comment:
Thoughts on moving this check to `_stake`? to reuse code?

Reply:
I was anticipating a question about this :)

I thought about applying it to _stake(), but preferred to validate directly on the external function interface, so it's "maximally easy" to for a reviewer check that each of the input arguments are validated as easy as possible, before going any further.

It's of course a fairly simple contract, so it should also be trivially verifiable to require on _stake() instead. That would save a single require() statement. I can see arguments for both, so am definitely open to making adjustments if there is a preference for one or the other.

Reply:
This isn't a gas savings consideration just a readability and also easier for someone in future to get this require statement "for free" if they use _stake() in the future. I don't think we should be concerned about a reviewer understanding the changes here since the overall diff is so small.